x-pack/qa/rolling-upgrade/src/test/resources/rest-api-spec/test/old_cluster/40_ml_datafeed_crud.yml

```diff
@@ -29,7 +30,34 @@
             "job_id":"old-cluster-datafeed-job",
             "indices":["airline-data"],
             "types":["response"],
-            "scroll_size": 2000
+            "scroll_size": 2000,
+            "aggregations": {
```

Comment:
Since the wire transport is quite different for `null` and non-`null` aggregations I think it would be good to have a second datafeed that doesn't use aggregations.  I will push another test to this PR soon.